internal/aks/create.go

```diff
@@ -128,6 +130,15 @@ func CreateOrUpdateCluster(ctx context.Context, cred *Credentials, clusterClient
 		}
 	}
 
+	addonProfiles := map[string]*containerservice.ManagedClusterAddonProfile{
+		"httpApplicationRouting": {
+			Enabled: spec.HTTPApplicationRouting,
+		},
+	}
+	if !hasHTTPApplicationRoutingSupport(spec) {
```

Comment:
some kind of log message might be nice since this field is just being ignored

Reply:
As I see this is also for China Azure cloud, probably this should be general information about potential limitation for China aks deployments.